### ISSUE TITLE
WebRTC crash: RTCDataChannel.readyState is not 'open'

### ISSUE THREAD

**Issue by dguenther** [iron-fish/ironfish#16]
I've seen this error come up in Discord several times:

```
(node:1) UnhandledPromiseRejectionWarning: InvalidStateError: RTCDataChannel.readyState is not 'open'
    at RTCDataChannel._send (<anonymous>)
    at RTCDataChannel.send (/usr/src/app/node_modules/wrtc/lib/index.js:49:8)
    at Peer.send (/usr/src/app/node_modules/simple-peer/index.js:239:19)
    at WebRtcConnection.send (/usr/src/app/node_modules/ironfish/build/src/network/peers/connections/webRtcConnection.js:37:23)
    at Peer.send (/usr/src/app/node_modules/ironfish/build/src/network/peers/peer.js:259:47)
    at PeerManager.sendTo (/usr/src/app/node_modules/ironfish/build/src/network/peers/peerManager.js:452:21)
    at PeerManager.handleSignalMessage (/usr/src/app/node_modules/ironfish/build/src/network/peers/peerManager.js:805:18)
    at PeerManager.handleMessage (/usr/src/app/node_modules/ironfish/build/src/network/peers/peerManager.js:554:18)
    at /usr/src/app/node_modules/ironfish/build/src/network/peers/peerManager.js:428:18
    at Event.emitAsync (/usr/src/app/node_modules/ironfish/build/src/event.js:61:39)
```

IRO-591

**Comment by razumv** [comment]
I have the same problem.
miner logs
![image](https://user-images.githubusercontent.com/42905535/113940991-0d48cd00-9807-11eb-80f0-594a7236e9ad.png)
node logs
![image](https://user-images.githubusercontent.com/42905535/113941071-281b4180-9807-11eb-8d98-911bc8a24927.png)

**Comment by artemrootman** [comment]
+1 same problem.

**Comment by lurenlym** [comment]
+1 same problem when using docker ubuntu

**Comment by drptbl** [comment]
Same issue on Ubuntu 18 with docker container.
```
{"log":"(node:1) UnhandledPromiseRejectionWarning: InvalidStateError: RTCDataChannel.readyState is not 'open'\n","stream":"stderr","time":"2021-04-10T09:16:25.926278422Z"}
{"log":"    at RTCDataChannel._send (\u003canonymous\u003e)\n","stream":"stderr","time":"2021-04-10T09:16:25.926280622Z"}
{"log":"    at RTCDataChannel.send (/usr/src/app/node_modules/wrtc/lib/index.js:49:8)\n","stream":"stderr","time":"2021-04-10T09:16:25.926282922Z"}
{"log":"    at Peer.send (/usr/src/app/node_modules/simple-peer/index.js:239:19)\n","stream":"stderr","time":"2021-04-10T09:16:25.926285122Z"}
{"log":"    at WebRtcConnection.send (/usr/src/app/node_modules/ironfish/build/src/network/peers/connections/webRtcConnection.js:37:23)\n","stream":"stderr","time":"2021-04-10T09:16:25.926287622Z"}
{"log":"    at Peer.send (/usr/src/app/node_modules/ironfish/build/src/network/peers/peer.js:259:47)\n","stream":"stderr","time":"2021-04-10T09:16:25.926290022Z"}
{"log":"    at GossipRouter.handle (/usr/src/app/node_modules/ironfish/build/src/network/messageRouters/gossip.js:73:24)\n","stream":"stderr","time":"2021-04-10T09:16:25.926292122Z"}
{"log":"    at PeerNetwork.handleMessage (/usr/src/app/node_modules/ironfish/build/src/network/peerNetwork.js:251:41)\n","stream":"stderr","time":"2021-04-10T09:16:25.926294322Z"}
{"log":"    at /usr/src/app/node_modules/ironfish/build/src/network/peerNetwork.js:50:63\n","stream":"stderr","time":"2021-04-10T09:16:25.926296622Z"}
{"log":"    at Event.emitAsync (/usr/src/app/node_modules/ironfish/build/src/event.js:61:39)\n","stream":"stderr","time":"2021-04-10T09:16:25.926298722Z"}
{"log":"    at Event.emit (/usr/src/app/node_modules/ironfish/build/src/event.js:52:19)\n","stream":"stderr","time":"2021-04-10T09:16:25.926300922Z"}
{"log":"(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 472)\n","stream":"stderr","time":"2021-04-10T09:16:25.926303122Z"}
```

**Comment by dguenther** [comment]
This should be fixed by #41 once it merges.
